opensync-gateway/src/main/java/com/telecominfraproject/wlan/opensync/ovsdb/dao/OvsdbDao.java

```diff
@@ -141,12 +141,18 @@
 
     @org.springframework.beans.factory.annotation.Value("${tip.wlan.ovsdb.listenPort:6640}")
     private int ovsdbListenPort;
```

Comment:
We should remove this variable, if it's not being used in the class and to avoid ambiguity.